Core/include/Acts/Seeding/SeedFilter.hpp

```diff
@@ -19,6 +19,17 @@
 #include <vector>
 
 namespace Acts {
+struct SeedConfQuantitiesConfig {
+	// number of minimum top SPs in seed confirmation
+	size_t nTopSeedConf = 0;
+  // number of high quality seeds in seed confirmation
+  int numQualitySeeds = 0;
+  // number of seeds that did not pass the quality confirmation but were still
+  // accepted, if quality confirmation is not used this is the total number of
+  // seeds
+  int numSeeds = 0;
```

Comment:
Would it make sense to pass other quantities in this struct as well?

Reply:
I could pass `std::vector<float>& invHelixDiameterVec` `std::vector<float>& impactParametersVec` and `float zOrigin`. But these are not necessarily related to the seed confirmation, so maybe I should rename the struct to something else

Reply:
As I wrote above, SeedingState could be possibly a good name encompassing more than confirmation. Another choice could be to have two structs?

Reply:
Most of the arguments were already included in a struct called `State` that I renamed to `SeedingState`. The other arguments are now included in one new struct called `SeedFilterState`.

Now we have 

` virtual void filterSeeds_2SpFixed(
      InternalSpacePoint<external_spacepoint_t>& bottomSP,
      InternalSpacePoint<external_spacepoint_t>& middleSP,
      std::vector<InternalSpacePoint<external_spacepoint_t>*>& topSpVec,
      SeedingState<external_spacepoint_t>& seedingState, SeedFilterState& seedFilterState,
      std::vector<std::pair<
          float, std::unique_ptr<const InternalSeed<external_spacepoint_t>>>>&
          outCont) const;`

We could also pass the `SeedingState` instead of most of the arguments but it will require some changes in the orthogonal seeding, which for some reason is not using the SeedingState. So I think it is better to do that in a different PR.

Reply:
That is ok for now. We are on good track. I agree that this MS is not place for it.